keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -1,32 +1,10 @@
-package com.netflix.spinnaker.keel.core.api
+package com.netflix.spinnaker.keel.api
 
-import com.fasterxml.jackson.annotation.JsonIgnore
-import com.fasterxml.jackson.annotation.JsonInclude
-import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY
-import com.fasterxml.jackson.annotation.JsonSubTypes
-import com.fasterxml.jackson.annotation.JsonSubTypes.Type
-import com.fasterxml.jackson.annotation.JsonTypeInfo
-import com.fasterxml.jackson.annotation.JsonTypeInfo.As
-import com.fasterxml.jackson.annotation.JsonTypeInfo.Id
-import com.fasterxml.jackson.annotation.JsonTypeName
-import com.netflix.spinnaker.keel.constraints.AllowedTimesConstraintEvaluator
 import java.time.Duration
 import java.time.Duration.ZERO
 
-@JsonTypeInfo(
-  use = Id.NAME,
-  include = As.PROPERTY,
-  property = "strategy"
-)
-@JsonSubTypes(
-  Type(RedBlack::class),
-  Type(Highlander::class)
-)
 sealed class ClusterDeployStrategy {
```

Comment:
Does this need to be changed to `abstract` now to allow plugins to extend it?

Reply:
I dunno about "now". The deploy strategy is really just something we pass to Orca. Any extensions to it are going to be a lot more wide-ranging than this PR.